dtsi.py

```diff
@@ -14,14 +18,52 @@ def generate_supplies(options: Options):
 	if options.regulator:
 		for r in options.regulator:
 			s += f"\t\t{r}-supply = <&...>;\n"
+		s += f"\n"
```

Comment:
No need for `f` if you just append a new line. But maybe we should just omit empty lines in the panel node? I guess you did this to separate the gpios or something, but this might result in a trailing line if there is no gpio at all, e.g.

```
panel {
	blah-supply = <&blah>;

};
```